INSTALL.md

```diff
@@ -24,56 +20,142 @@
 ## Installation
 
 ### Deploy the Container
-1. Connect to the RPi, via SSH for example.
-2. 
-    a. Clone the repo.
-
- **OR**
-
-2.
-    b. Create a `tesla_ble_mqtt_docker` folder in your user directory: `cd ~ && mkdir tesla_ble_mqtt_docker && cd $_.`
-
-    Add a docker-compose.yml file: `nano docker-compose.yml`.
-4. Adjust `docker-compose.yml` to your needs. Pay attention to the environment variables :
+There are various methods to deploy the container, I describe the main ones below:
+#### Deploy using Command Line ####
+1. Create a tesla_ble_mqtt_docker folder in your user directory and change directory into it:
    ```yaml
-   environment:
-      - TZ='Europe/London'
-      - TESLA_VIN =<tesla VIN> # Can be found in the App.
-      - MQTT_IP=<IP of the MQTT broker> # In most cases the local IP of HA.
-      - MQTT_PORT=1883 # 1883 is the default port, do not modify unless you specified another.
-      - MQTT_USER=<MQTT username> # Use what's specified in the MQTT broker on HA.
-      - MQTT_PWD=<MQTT password> # Use what's specified in the MQTT broker on HA.
+   cd ~ 
+   mkdir tesla_ble_mqtt_docker 
+   cd tesla_ble_mqtt_docker
    ```
-6. Save and exit.
-7. Create the Docker volume: `docker volume create tesla_ble_mqtt`
-8. Start the container: `docker-compose up -d`
-
-    Check the logs in Portainer (or other). They should look like this:
+2. Download docker-compose.yml and stack.env from the github repository:
+   ```yaml
+   curl -O https://raw.githubusercontent.com/tesla-local-control/tesla_ble_mqtt_docker/main/docker-compose.yml
+   curl -O https://raw.githubusercontent.com/tesla-local-control/tesla_ble_mqtt_docker/main/stack.env
+   ```
+3. Check docker-compose.yml contents are suitable for your needs. It will be ok as is for most people
+4. Update the environment variables in stack.env according to your needs. As a minimum enter the VIN of your car, and the connection details for your MQTT server. If you want BLE detection enter the BLE MAC address of the car (see below for instructions on how to find this TODO):
+   ```yaml
+   # Optional for car presence detection; If multiple cars, separate with , or | or white space
+   #
+   BLE_MAC_LIST=
+   # Default 5 (seconds)
+   #
+   BLE_CMD_RETRY_DELAY=
+   ### Default false
+   #
+   DEBUG=

```

Comment:
maybe move all the "optional" variables, ie. the one with default values, at the end of the list?

Reply:
I just copied the env (stack.env) file as it was created by Pierre (removing the blank lines). I think you are right though So I will amend both INSTALL.md and stack.env accordingly

Reply:
Reordered it slightly to keep logical groupings but have Mandatory items at the top